jaeger-core/src/test/java/com/uber/jaeger/SpanTest.java

```diff
@@ -21,9 +21,7 @@
  */
 package com.uber.jaeger;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
you need to fix your IDE formatter. I think readme has links to IntelliJ settings